typescript/sdk/src/core/HyperlaneCoreChecker.ts

```diff
@@ -118,17 +119,13 @@ export class HyperlaneCoreChecker extends HyperlaneAppChecker<
 
     const actualIsm = await mailbox.defaultIsm();
     const config = this.configMap[chain];
-    /*
-    TODO: Add this back in once the new ISM factories are adopted
-    const matches = await moduleMatches(
+    const matches = await moduleMatchesConfig(
```

Comment:
👍

Reply:
will revert and address in a separate PR